### PR TITLE
fix disabled state interactions

### PR DESCRIPTION
**Pull request by JafarSaifullah**
resolves https://github.com/optimizely-axiom/optiaxiom/issues/67

### CODE REVIEW

File: packages/react/src/button/Button.css.ts [hunk 1]
```diff
@@ -69,51 +69,51 @@ export const button = recipe({
...
       },
       primary: {
         "@layer": {
           [layers.axiom]: {
-            vars: {
-              [accentColorVar]: theme.colors["bg.brand.solid"],
-              [solidAccentColorVar]: theme.colors["bg.brand.solid.hover"],
-              [subtleAccentColorVar]: theme.colors["bg.brand.subtle"],
-            },
-
```
Comment (MunifTanjim) [inline packages/react/src/button/Button.css.ts:94]: Can we revert these unnecessary changes?

Comment (JafarSaifullah) [reply packages/react/src/button/Button.css.ts]: addressed bhai

**Review by MunifTanjim — APPROVED** [review]
